src/applications/personalization/profile-2/components/ProfileInfoTable.jsx

```diff
@@ -51,35 +52,41 @@ const ProfileInfoTable = ({
   const tableRowDataClasses = ['vads-u-margin--0'];
 
   return (
-    <div className={tableClasses.join(' ')} data-field-name={fieldName}>
+    <section className={tableClasses.join(' ')} data-field-name={fieldName}>
       {title && (
         <h3 className={[...titleClasses, ...titleClassesMedium].join(' ')}>
           {title}
         </h3>
       )}
-      {data
-        .map(element => (dataTransformer ? dataTransformer(element) : element))
-        .map((row, index) => (
-          <div
-            key={index}
-            className={`'table-row' ${[
-              'table-row',
-              ...tableRowClasses,
-              ...tableRowClassesMedium,
-            ].join(' ')}`}
-          >
-            <h4
-              className={[
-                ...tableRowTitleClasses,
-                ...tableRowTitleMediumClasses,
-              ].join(' ')}
+      {/* This syntax is kind of ugly. But it's simply adding `role="list"` if `list` is truthy` */}
+      <dl className="vads-u-margin--0" {...{ ...(list && { role: 'list' }) }}>
```

Comment:
switching the markup to use a `dl` for the entire table contents, and then using a `dt` for the row's "title" and a `dd` for the row's "value"

Reply:
👀 woah